atc/db/resource_check_rate_limiter_test.go

```diff
@@ -56,14 +61,15 @@ var _ = Describe("ResourceCheckRateLimiter", func() {
 	}
 
 	createCheckable := func() {
-		config, err := resourceConfigFactory.FindOrCreateResourceConfig(
-			defaultWorkerResourceType.Type,
-			atc.Source{"some": "source", "count": checkableCount},
-			atc.VersionedResourceTypes{},
-		)
+		es := dbConn.EncryptionStrategy()
+		encryptedPayload, nonce, err := es.Encrypt([]byte("{}"))
 		Expect(err).ToNot(HaveOccurred())
 
-		_, err = config.FindOrCreateScope(nil)
+		_, err = psql.Insert("resources").
```

Comment:
can we change this to use the `dbtest` package instead? it's best to avoid running raw queries since they make the tests and schema hard to maintain.

something like this should work:

```go
var scenario *dbtest.Scenario

BeforeEach(func() {
  scenario = dbtest.Scenario() // no initial setup
})

createCheckable := func() {
  checkableCount++

  var resources atc.ResourceConfigs
  for i := 0; i < checkableCount; i++ {
    resources = append(resources, atc.ResourceConfig{
      Name: fmt.Sprintf("resource-%d", i)),
      Type: dbtest.BaseResourceType,
      Source: atc.Source{"some": "source"},
    })
  }

  scenario.Run(builder.WithPipeline(atc.Config{
    Resources: resources,
  })
}
```

Reply:
Fixed.